### PR TITLE
Add PHP Standards Document

### PR DESCRIPTION
**Pull request by wheatleyjj**

### CODE REVIEW

File: platforms/web/php/standards.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+*   [Amendments](#amendments)
+    *   [Variables](#variables)
+    *   [Methods](#methods)
+*   [Adhering to standards](#adhering)
+    *   [PhpStorm + PHP Code Sniffer](#codesniffer)
+    *   [PhpStorm + PHP-CS-Fixer](#csfixer)
+
+
+
+<h3 id="introduction">Introduction</h3>
```
Comment (dextermb) [inline platforms/web/php/standards.md]: How come you are using html tags over markdown? The majority of the file seems to be markdown?

Comment (wheatleyjj) [reply platforms/web/php/standards.md]: After wanting to build a table of contents I checked online for an example for syntax and it used header tags to enable the use of the id field for linking the contents to the headers.

Comment (dextermb) [reply platforms/web/php/standards.md]: When you commit file changes in markdown, using headers, GitHub auto generate links. The annoying thing is that it is not shown in the preview while editing online

Comment (dextermb) [reply platforms/web/php/standards.md]: Checkout the source for the helpers README:
https://github.com/langleyfoxall/helpers-laravel/blob/master/README.md

Here has all headers using markdown, linked and has duplicate headers so you can see how they get numbered too.

File: platforms/web/php/standards.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        *   [PSR-1](#php)   
+        *   [PSR-2](#php)   
+    *   [Laravel](#laravel)
+    *   [RESTful API's](#restful-api's)
+*   [Amendments](#amendments)
+    *   [Variables](#variables)
+    *   [Methods](#methods)
+*   [Adhering to standards](#adhering-to-standards)
+    *   [PhpStorm + PHP Code Sniffer](#phpstorm-+-php-code-sniffer)
+    *   [PhpStorm + PHP-CS-Fixer](#phpstorm-+-php-cs-fixer)
```
Comment (dextermb) [inline platforms/web/php/standards.md]: Seems like this link becomes: `phpstorm--php-code-sniffer`

**Pull request merged**